__tests__/app.test.js

```diff
@@ -325,4 +325,32 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("/api/comments", () => {
+    describe("/api/comments/:comment_id", () => {
+      test("DELETE: 204 - responds by deleteing a comment via its comment_id and returning no content", () => {
+        return request(app)
+          .delete("/api/comments/1")
+          .expect(204)
+          .then((response) => {
+            expect(response.body).toEqual({});
```

Comment:
A 204 comes with no body by default - so you could leave out line 335 as long as the 204 assertion is passing.